composer.json

```diff
@@ -24,7 +24,9 @@
     ],
     "require": {
         "php": "^7.4|^8.0|^8.1|^8.2|^8.3",
-        "ext-json": "*"
+        "ext-json": "*",
+        "psr/container": "^2.0",
```

Comment:
I know you have this in here for one single comment. I don't know if adding a dependency is worth a comment. Given the stuff that's happened with xz, I'm scrutinizing every dependency I can.